From wfp5p@virginia.edu  Mon Jul 27 14:10:38 2009
From: Bill Pemberton <wfp5p@virginia.edu>
Date: Mon, 27 Jul 2009 16:47:37 -0400
Subject: Staging: hv: vmbus_drv.c: remove unused structs
To: greg@kroah.com
Cc: hjanssen@microsoft.com
Message-ID: <1248727665-23969-16-git-send-email-wfp5p@virginia.edu>


vmbus_ctl_table_hdr, vmbus_dev_ctl_table, vmbus_ctl_table, and
vmus_root_ctl_table were never used.  This removes them.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/vmbus_drv.c |   28 ----------------------------
 1 file changed, 28 deletions(-)

--- a/drivers/staging/hv/vmbus_drv.c
+++ b/drivers/staging/hv/vmbus_drv.c
@@ -101,34 +101,6 @@ static int vmbus_irq = VMBUS_IRQ;
 
 /* Setup /proc/sys/bus/vmbus/vmbus_loglevel */
 /* Allow usage of sysctl cmd to set the logging level */
-static struct ctl_table_header *vmbus_ctl_table_hdr;
-
-static ctl_table vmbus_dev_ctl_table[] = {
-	{ .ctl_name	= 8461,
-	  .procname	= "vmbus_loglevel",
-	  .data		= &vmbus_loglevel,
-	  .maxlen	= sizeof(vmbus_loglevel),
-	  .mode		= 0644,
-	  .proc_handler	= &proc_dointvec },
-	{ }
-};
-
-static ctl_table vmbus_ctl_table[] = {
-	{ .ctl_name	= CTL_DEV,
-	  .procname	= "vmbus",
-	  .mode		= 0555,
-	  .child	= vmbus_dev_ctl_table },
-	{ }
-};
-
-static ctl_table vmus_root_ctl_table[] = {
-	{ .ctl_name	= CTL_BUS,
-	  .procname	= "bus",
-	  .mode		= 0555,
-	  .child	= vmbus_ctl_table },
-	{ }
-};
-
 
 /* Set up per device attributes in /sys/bus/vmbus/devices/<bus device> */
 
